app/controllers/support_interface/identity_controller.rb

```diff
@@ -27,7 +29,20 @@ def confirm
     def callback
       flash[:success] = "You have completed a simulated Identity journey"
 
-      redirect_to support_interface_identity_path
+      redirect_to support_interface_identity_simulate_path
+    end
+
+    def users
+      @all_users ||= IdentityApi.get_users
+      @total = @all_users.size
+      @pagy, @users = pagy_array(@all_users)
+    rescue IdentityApi::ApiError,
+           Faraday::ConnectionFailed,
+           Faraday::TimeoutError,
+           ActionController::ActionControllerError => e
+      Sentry.capture_exception(e)
+      # Do not redirect back to Get An Identity
+      render "errors/internal_server_error", status: :internal_server_error
     end
```

Comment:
@malcolmbaig yes good suggestion, I think this would make sense as the changes that you are already making include show page using a `UsersController`.